chains/evm/calls/consts/consts.go

```diff
@@ -1,7 +1,10 @@
 package consts
 
+import "time"
+
 const DefaultGasLimit = 2000000
 const DefaultDeployGasLimit = 6000000
 const DefaultGasPrice = 20000000000
 const DefaultGasMultiplier = 1
 const DefaultBlockConfirmations = 10
+const DefaultBlockRetryInterval = time.Duration(5) * time.Second
```

Comment:
Just a fun fact, you don't need to wrap 5 in `time.Duration`.

`const DefaultBlockRetryInterval = 5 * time.Second` would behave the same.